prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -1,54 +1,87 @@
+
+* `accountId`: The accountId is used to define the accountID and the "account settings" used for this app in Prebid Server. These allow your Prebid Server to lookup account details and attributes like timeout and price granularity.
+
+* `adServer`: This parameter specifies the ad server you are using. It is an enum value of the type `Prebid.AdServer`. The possible values include `Prebid.AdServer.DFP` for Google Ad Manager and `Prebid.AdServer.MOPub` for MoPub.
+
+* `host`: If you are using a managed Prebid server service, the SDK is coded with enums containing the URLs for those services, for example `Prebid.Host.RUBICON` and `Prebid.Host.APPNEXUS`. Other Prebid Server managed services exist that don't have enum values. You can work with them by entering the appropriate URL. If you are using a self-hosted server, you should enter the URL to your server.
+
+For example:
+
+```java
+Prebid.init(getApplicationContext(), adUnits, "INSERT-ACCOUNT-ID-HERE", Prebid.AdServer.DFP, "https://mypbs.example.org");
+```
+
+## Define first party data and set up adUnits
+
+When using a custom or 3rd-party ad server, you must first get the targeting keys using the `fetchDemand` method. This function provides the bidder key/values (targeting keys). You can then pass these targeting keys to the ad server of your choice.
+
+You will need to perform the following actions:
+
+* Call `fetchDemand` with extended `targetingDict` callback.
+* Retrieve the targeting keys from `fetchDemand`.
+* Convert targeting keys into the format used by your ad server.
+* Pass the targeting keys to your ad server.
+* Render the ad with the Prebid Universal Creative or custom renderer.
+
+This approach is available for the ad formats listed below. For detailed integration steps, click on the link corresponding to the ad format:
+
+* [Display Banner](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#banner-api)  via `BannerAdUnit`
+* [Video Banner](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#video-banner-outstream-video) and Instream Video via `VideoAdUnit`
+* [Display Interstitial](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#interstitial-api) via `InterstitialAdUnit`
+* [Video Interstitial](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#video-interstitial) via `InterstitialAdUnit`
+* [Rewarded Video](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#rewarded-video-api) via `RewardedVideoAdUnit`
+* [Native Banner](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#native-api) via `NativeAdUnit`
+
+You will also need to use the `fetchDemand` function with the following signature:
```

Comment:
Again, this feels backwards... why are we talking about fetchDemand before we've created any adunits? Maybe the intention is that the links just above are what people are supposed to follow. 

If so, then I would suggest:
- provide a simple banner example so there's at least a complete "hello world" flow
- that example should contain some minimal First Party Data because it's so important nowadays.
- Push the fetchDemand stuff to the next section. This section is for setting up.